src/dune_rules/install_rules.ml

```diff
@@ -97,9 +97,10 @@ end = struct
       >>| Modules.With_vlib.modules
       >>| Option.some
     and+ foreign_archives =
-      match Lib_info.virtual_ lib with
-      | false -> Memo.return (Mode.Map.Multi.to_flat_list @@ Lib_info.foreign_archives lib)
-      | true ->
+      match Lib_info.kind lib with
+      | Dune_file _ ->
+        Memo.return (Mode.Map.Multi.to_flat_list @@ Lib_info.foreign_archives lib)
+      | Parameter | Virtual ->
```

Comment:
This should be empty for `Parameter`. I don't think we allow stubs for parameters.

Reply:
Oh right, thanks, done!